README.md

```diff
@@ -36,15 +36,13 @@ If you already have a StatsD client running, see the STATSD_URL configuration op
 
 ```
 heroku buildpacks:add heroku/go
-heroku buildpacks:add --index 1 https://github.com/miketheman/heroku-buildpack-datadog.git
 heroku config:set HEROKU_APP_NAME=$(heroku apps:info|grep ===|cut -d' ' -f2)
```

Comment:
Where is that used?

Reply:
No where I believe, I'll cleanup the doc.